<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mul_thread_qtClass</class>
 <widget class="QMainWindow" name="mul_thread_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mul_thread_qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>851</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>195</width>
       <height>541</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>195</width>
       <height>541</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>文件路径选择</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>参数调整</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>开始转化</string>
      </property>
     </item>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>650</width>
       <height>541</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>541</height>
      </size>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>541</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>输入视频：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>...(mp4格式)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>输出视频：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>...(选择文件夹)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="startPlaybut">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>170</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>491</width>
         <height>371</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QPushButton" name="stopPlaybut">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>563</width>
         <height>501</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>561</width>
            <height>111</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>561</width>
            <height>111</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>0</y>
             <width>197</width>
             <height>103</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="3" column="1">
               <widget class="QSpinBox" name="spinBox_2"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>归一化</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>模糊</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>线程数目</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_3"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="minimumSize">
               <size>
                <width>61</width>
                <height>51</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>61</width>
                <height>51</height>
               </size>
              </property>
              <property name="text">
               <string>测试</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>561</width>
            <height>381</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>561</width>
            <height>381</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>218</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>开始转化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mul_thread_qt.qrc"/>
 </resources>
 <connections/>
</ui>
